### DISCUSSION TITLE
Few clients disconnect and connect after 1 seconds... - not always the same...

### DISCUSSION THREAD

**Discussion by abixadamj** [epoptes/epoptes#157] [Q&A]
I have a school with LAN:
1 router (with DHCP) - 1 dumb 24port switch -> 20 PCs and a server
On server Ubuntu 20.04 and LTSP.
root@myslowice-zsp:~# ltsp info
LTSP version: 22.01-1~ubuntu20.04.1
SERVER XSESSIONS: mate.desktop
IMAGES:
x86_64

Sometimes (usually when more than 10 clients run Firefox) few random client suddenly dosconnects from epoptes and connects after1-2 seconds.
I t is strange situatiuon, I dont't have any idea, what to check....
![Zrzut ekranu z 2022-05-13 14-14-36](https://user-images.githubusercontent.com/5088643/168284129-9f4189b5-8e12-41c7-be28-cc4ec5d8c0d2.png)

The only clue I can think of is NFS traffic... but there are only 2 exported directories:

```
root@myslowice-zsp:/srv# du -sh apps uczniowie/
1,2G	apps
18M	uczniowie/
```

Can this be a reason ?

**Comment by alkisg** [comment]
Indeed heavy network traffic may cause thumbshots to fail reaching the server.
The network traffic can be NFS (/srv/ltsp, /srv/uczniowie) or SSHFS (/home).
Unfortunately to keep the client requirements low (1 MB RAM) it's written in shell instead of Python, so doing fancy stuff isn't possible.
